src/table/components/TableTotals.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import TableRow from '@mui/material/TableRow';
+import TableCell from '@mui/material/TableCell';
+import { getHeaderStyle, showTotalsAtTop } from '../utils/styling-utils';
+import { headHandleKeyPress } from '../utils/handle-key-press';
+import { handleClickToFocusHead } from '../utils/handle-accessibility';
+
+function TableTotals({
+  rootElement,
+  tableData,
+  theme,
+  layout,
+  changeSortOrder,
+  constraints,
+  translator,
+  setFocusedCellCoord,
+  keyboard,
+}) {
+  const { columns, paginationNeeded } = tableData;
+
+  const headerStyle = useMemo(() => getHeaderStyle(layout, theme), [layout, theme.name()]);
+  const totalCellStyle = () => {
+    headerStyle.borderColor = '#a6a6a6';
+    showTotalsAtTop(layout)
+      ? (headerStyle.borderWidth = '0px 1px 2px 0px')
+      : (headerStyle.borderWidth = '2px 1px 1px 0px');
+    return [headerStyle, { fontWeight: 'bold' }];
+  };
+  const headRowStyle = {
+    '& :last-child': {
+      borderRight: paginationNeeded && 0,
+    },
+    'th:first-of-type': {
+      borderLeft: !paginationNeeded && '1px solid rgb(217, 217, 217)',
+    },
+  };
+
+  const totalRowStyle = () => {
+    if (showTotalsAtTop(layout)) return [headRowStyle, { position: 'sticky', top: 35 }];
+    return [headRowStyle, { position: 'sticky', bottom: 0 }];
```

Comment:
Why an array with two objects? btw do you need to add sticky here? how does the normal header stay sticky?